avo-hq/avo
# Made it possible to customise the placeholder image

# Description


Makes it possible to replace the avocado image used for placeholders with a custom image.

Fixes #1243

# Checklist:




## Manual review steps


1. Set the `placeholder` value in the branding config with the path to an image
2. Create a new resource that has a grid but don't set a cover image
3. Check that the new placeholder image is used
4. Remove the placeholder value from the branding and restart
5. Confirm that the placeholder changes back to an avocado.

Manual reviewer: please leave a comment with output from the test if that's the case.
